app/views/hyrax/collections/_sort_and_per_page.html.erb

```diff
@@ -1,24 +1,17 @@
-<div class="batch-info">
```

Comment:
These were used for this style: https://github.com/samvera/hyrax/blob/8cd726b1d16fa1b85eec02d2fbdaa2f71c31178a/app/assets/stylesheets/hyrax/_file-listing.scss#L70  is it now orphaned?

Reply:
No, the style is still used on these pages:

https://github.com/samvera/hyrax/blob/0b2536d2cd1531218d97305afd535f0af98743cb/app/views/hyrax/dashboard/collections/_sort_and_per_page.html.erb

https://github.com/samvera/hyrax/blob/0b2536d2cd1531218d97305afd535f0af98743cb/app/views/hyrax/my/_search_header.html.erb